shared/general/labels.md

```diff
@@ -1,15 +1,18 @@
-Label | Default | Description | Valid from
---- | --- | --- | ---
-io.{{ $names.company.short }}.features.balena-socket | false | Bind mounts the balena container engine socket into the container. | v7.22.0
-io.{{ $names.company.short }}.features.dbus | false | Bind mounts the host OS dbus into the container using `/run/dbus:/host/run/dbus`. | v7.22.0
-io.{{ $names.company.short }}.features.sysfs | false | Bind mounts the host OS `/sys` into the container. | v10.8.0
-io.{{ $names.company.short }}.features.procfs | false | Bind mounts the host OS `/proc` into the container. | v10.8.0
-io.{{ $names.company.short }}.features.kernel-modules | false | Bind mounts the host OS `/lib/modules` into the container (i.e. `/lib/modules:/lib/modules`). | v7.22.0
-io.{{ $names.company.short }}.features.firmware | false | Bind mounts the host OS `/lib/firmware` into the container. | v7.22.0
-io.{{ $names.company.short }}.features.supervisor-api | false | Ensures that `{{ $names.company.allCaps }}_SUPERVISOR_HOST`, `{{ $names.company.allCaps }}_SUPERVISOR_PORT`, `{{ $names.company.allCaps }}_SUPERVISOR_ADDRESS`, and `{{ $names.company.allCaps }}_SUPERVISOR_API_KEY` are added to the container environment variables, so the supervisor API can be used. | v7.22.0
-io.{{ $names.company.short }}.features.{{ $names.company.short }}-api | false | When enabled, it will make sure that `{{ $names.company.allCaps }}_API_KEY` is added to the container environment variables. | v7.22.0
-io.{{ $names.company.short }}.update.strategy | download-then-kill | Set the application [update strategy][update-strategy]. | v7.22.0
-io.{{ $names.company.short }}.update.handover-timeout | 60000 | Time, in milliseconds, before an old container is automatically killed. Only used with the `hand-over` [update strategy][hand-over]. | v7.22.0
+Label | Default | Description | Supervisor | balenaOS*
```

Comment:
This file is part of the following docs page:
https://www.balena.io/docs/learn/develop/multicontainer/#labels

The table had a column named "Valid from" with version numbers, but I thought it wasn't super clear "versions of what" they were. They were balenaSupervisor versions. But for most ends users, I also though that a balenaOS version would be more useful than a supervisor versions, as users need to know which balenaOS version to download and they will not normally be updating the supervisor independently of balenaOS. I thought of having only the balenaOS version, but this would be a bit... imprecise. I've settled for adding both versions, renaming the columns and adding a footnote.

This is completely of my own reasoning, and not yet discussed with anyone else. Feel free to push back if you are not completely comfortable with the PR!